Library/Homebrew/tap.rb

```diff
@@ -1352,7 +1352,12 @@ def core_cask_tap?
 
   sig { params(token: String).returns(Pathname) }
   def new_cask_path(token)
-    cask_subdir = token[0].to_s
+    cask_subdir = if token.start_with?("font")
+      "fonts/font-#{token[5]}"
```

Comment:
```suggestion
    cask_subdir = if token.start_with?("font-")
      "fonts/#{token.delete_prefix("font-")}"
```
Seems nicer.

Not sure how you deal with non-cask fonts whose tokens start with `font-`, though (e.g. `font-finagler`, `font-smoothing-adjuster`). I suppose you could not care, or bake in exceptions here or as some sort of exception list in the cask repo.

Reply:
That's much better. Thanks. Will see if anyone else has a clever idea for how to handle the issue with those other casks.

Reply:
Looking at the CI run, this doesn't produce the expected result. We are after `Casks/fonts/font-a/font-a-example.rb`, `Casks/fonts/font-b/font-b-example.rb`, etc. Deleting the prefix gives everything it's own folder under `Casks/fonts` which is not what we want.